docs/faq.asciidoc

```diff
@@ -44,6 +44,26 @@ It also respects other agents attached to your application at the same time.
 === Do I need to re-compile my application?
 No
 
+[float]
+[[recommended-sampling-rate]]
+=== What is the recommended sampling rate?
+There is no one-size-fits-all answer to an ideal sampling rate.
+Sampling comes down to your preferences and your application.
+The more you want to sample, the more network bandwidth and disk space you'll need.
+
+It's important to note that the latency of an application won't be affected much by the agent (in the order of single-digit microseconds),
+even if you sample at 100%.
+However, the background reporter thread has some work to do for serializing and gzipping events.
+If your application is not CPU bound, this shouldn't matter much.
+
```

Comment:
I think adding the last part is good as well:
>Note that if the APM Server can't handle all the events, the agent will drop data to not crash your application. It will then also not serialize and gzip the events.

Reply:
Added!